src/main/settings.py

```diff
@@ -118,6 +119,15 @@
 STATIC_URL = '/mensen/static/'
 STATIC_ROOT = '/static/'
 
+INGRESS_CONSUMER_CLASSES = [
+    'centralerekenapplicatie_v1.ingress_parser.MetricParser',
+    'telcameras_v2.ingress_parser.TelcameraParser',
+    'telcameras_v3.ingress_parser.TelcameraParser'
```

Comment:
Waarom geven we hier eigenlijk niet gewoon de class in, ipv een string naar de class?

Reply:
Je wilt voorkomen dat wanneer je app start en de settings gelezen worden, er te veel classes geimport worden. In settings zitten daarom nooit expliciete classes, altijd een path naar een class (denk aan database engines, authentication en permission classes in drf, cache backend settings, email backends, etc).